blocks/basic/test/qa_BasicKnownBlocks.cpp

```diff
@@ -0,0 +1,27 @@
+#include <fmt/format.h>
+#include <fmt/ranges.h>
+
+#include <cassert>
+
+#include <gnuradio-4.0/basic/all.hpp>
+
+int
+main() {
+    using namespace std::string_literals;
+
+    auto known = gr::globalBlockRegistry().knownBlocks();
+    std::ranges::sort(known);
+
+    std::vector<std::string> desired{
+        //
+        "builtin_counter"s,              //
+        "builtin_multiply"s,             //
+        "gr::basic::DataSink"s,          //
+        "gr::basic::FunctionGenerator"s, //
+        "gr::basic::Selector"s,          //
+        "gr::basic::SignalGenerator"s    //
+    };
+    std::ranges::sort(desired);
```

Comment:
This is a brittle test and will fail whenever someone adds a new Block to `gr::globalBlockRegistry()`. Must be changed to something more robust and/or test a subset/intersection.

Reply:
OK. mea culpa. Assumed that `std::ranges::includes` would required both sorted and contiguous, which it doesn't

https://compiler-explorer.com/z/Gz89frx8n